subcmds/report.go

```diff
@@ -179,10 +182,16 @@ func (p *ReportCmd) SetFlags(f *flag.FlagSet) {
 
 	f.StringVar(&config.Conf.TrivyCacheDBDir, "trivy-cachedb-dir",
 		cache.DefaultDir(), "/path/to/dir")
-	f.StringVar(&config.Conf.TrivyJavaDBRepository, "trivy-java-db-repository",
-		trivyFlag.JavaDBRepositoryFlag.Default, "Trivy Java DB Repository")
-	f.BoolVar(&config.Conf.TrivySkipJavaDBUpdate, "trivy-skip-java-db-update",
-		false, "Skip Trivy Java DB Update")
+
+	config.Conf.TrivyOpts.TrivyDBRepositories = []string{trivydb.DefaultGHCRRepository}
+	dbRepos := stringArrayFlag{target: &config.Conf.TrivyOpts.TrivyDBRepositories}
+	f.Var(&dbRepos, "trivy-db-repository", fmt.Sprintf("Trivy DB Repository in a comma-separated list (default %s)", trivydb.DefaultGHCRRepository))
+
+	config.Conf.TrivyOpts.TrivyJavaDBRepositories = []string{trivyjavadb.DefaultGHCRRepository}
+	javaDBRepos := stringArrayFlag{target: &config.Conf.TrivyOpts.TrivyJavaDBRepositories}
+	f.Var(&javaDBRepos, "trivy-java-db-repository", fmt.Sprintf("Trivy Java DB Repository in a comma-separated list (default %s)", trivyjavadb.DefaultGHCRRepository))
+
+	f.BoolVar(&config.Conf.TrivySkipJavaDBUpdate, "trivy-skip-java-db-update", false, "Skip Trivy Java DB Update")
```

Comment:
Is it necessary to add the same flag here as well?
https://github.com/future-architect/vuls/blob/e9a509aaeebc30aa99742e623822d1cf85b6dfbd/subcmds/report_windows.go#L174-L176

Reply:
It's probably historical reason (?), TrivyCacheDBDir is defaulted to "<cache-dir>/trivy"(Wasn't there java DB yet? I don't know). It's the parent directory of current vulnerability db and javadb. The name is not good...

Reply:
So,... complete treatment is to add TrivyCacheDir (which points "<cache-dir>/trivy" option and deprecate TrivyCacheDBDir ? 
Another way is to add "vulnerability" to vulnerability db option name, e.g. TrivyVulnerabilityDBRepositories ("--trivy-vulnerability-db-repository" in command line flag). I feel it's too long, isn't it?

Reply:
What this comment is trying to say is that the vuls report command built on Windows does not seem to have `--trivy-db-repository`.

Reply:
Ooops! I forgot about it completely... Thanks!

Reply:
Done at https://github.com/future-architect/vuls/compare/e9a509aaeebc30aa99742e623822d1cf85b6dfbd..c66d7d2923091dd55e227cf82ec93b6e9d5bea8d#diff-696031422e2add743d99a7aa2a1b2caecb0effc52500005553a89e3e440dbacf